packages/@o3r-training/tools/project.json

```
@@ -0,0 +1,67 @@
+{
+  "name": "training-tools",
+  "$schema": "https://raw.githubusercontent.com/nrwl/nx/master/packages/nx/schemas/project-schema.json",
+  "projectType": "library",
+  "sourceRoot": "packages/@o3r-training/tools/src",
+  "prefix": "o3r",
+  "targets": {
+    "build": {
+      "executor": "nx:run-script",
+      "outputs": ["{projectRoot}/dist/package.json"],
+      "options": {
+        "script": "postbuild"
+      },
+      "dependsOn": [
+        "^build",
+        "compile",
+        "build-cli"
+      ]
+    },
+    "prepare-build-builders": {
+      "executor": "nx:run-script",
+      "options": {
+        "script": "prepare:build:builders"
+      }
+    },
+    "build-cli": {
+      "executor": "nx:run-script",
+      "options": {
+        "script": "build:cli"
+      }
+    },
+    "compile": {
+      "executor": "nx:run-script",
+      "options": {
+        "script": "build:source"
+      }
+    },
+    "lint": {
+      "options": {
+        "eslintConfig": "packages/@o3r-training/tools/.eslintrc.js",
+        "lintFilePatterns": [
+          "packages/@o3r-training/tools/src/**/*.ts",
+          "packages/@o3r-training/tools/package.json"
+        ]
+      }
+    },
+    "test": {
+      "executor": "@nx/jest:jest",
+      "options": {
+        "jestConfig": "packages/@o3r-training/tools/jest.config.js"
+      }
+    },
+    "prepare-publish": {
+      "executor": "nx:run-script",
+      "options": {
+        "script": "prepare:publish"
+      }
+    },
+    "publish": {
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "npm publish packages/@o3r-training/tools/dist"
+      }
+    }
+  },
+  "tags": []
```

```suggestion
  "tags": ["showcase"]
```
To not publish the package to npmjs

---

The tag to prevent the publish is actually `private`
```suggestion
  "tags": ["private", "showcase"]
```